network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css

```diff
@@ -0,0 +1,42 @@
+/*
+  To address bug from Wagtail v3
+  See context: https://github.com/mozilla/foundation.mozilla.org/issues/10192
+*/
+body#wagtail
+  .sidebar-panel
+  .c-page-explorer__see-more
+  svg.icon.icon-arrow-right {
+  width: 20px;
+  height: 20px;
+  position: relative;
+  top: 5px;
+}
+
+/*
+  To address bug from Wagtail v3
+  See context: https://github.com/mozilla/foundation.mozilla.org/issues/10194
+*/
+
+body#wagtail
+  main#main
+  form
+  ul
+  li
+  .field
+  .c-dropdown.t-inverted.c-dropdown--large
+  a {
```

Comment:
I'm using very specific selector so there's less chance of these styling rules affecting elements that we are unaware of.